airbyte-integrations/connectors/destination-dynamodb/src/main/java/io/airbyte/integrations/destination/dynamodb/DynamodbOutputTableHelper.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.dynamodb;
+
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+import io.airbyte.protocol.models.AirbyteStream;
+import java.util.LinkedList;
+import java.util.List;
+
+public class DynamodbOutputTableHelper {
+
+  public static String getOutputTableName(String tableName, AirbyteStream stream) {
+    return getOutputTableName(tableName, stream.getNamespace(), stream.getName());
+  }
+
+  public static String getOutputTableName(String tableName, String namespace, String streamName) {
+    List<String> paths = new LinkedList<>();
+
+    if (tableName != null) {
+      paths.add(tableName);
+    }
+    if (namespace != null) {
+      paths.add(new ExtendedNameTransformer().convertStreamName(namespace));
+    }
+    if (streamName != null) {
+      paths.add(new ExtendedNameTransformer().convertStreamName(streamName));
+    }
+
+    return String.join("_", paths);
```

Comment:
This should follow the convention from [here](https://docs.airbyte.io/understanding-airbyte/namespaces#examples).

The custom_namespace dont custom shouldn't start with the table name informed in spec.json. This is possible?
![Screenshot from 2021-08-24 21-03-59](https://user-images.githubusercontent.com/5154322/130705322-1363495a-3c9c-48b2-9868-839628a8d30b.png)

Reply:
We changed the sequence for the table name in the new commit.